google/cloud/storage/testing/storage_integration_test.cc

```diff
@@ -50,13 +51,16 @@ StorageIntegrationTest::MakeIntegrationTestClient(
 
 std::unique_ptr<BackoffPolicy> StorageIntegrationTest::TestBackoffPolicy() {
   std::chrono::milliseconds initial_delay(std::chrono::seconds(1));
+  auto constexpr kShortDelayForTestbench = std::chrono::milliseconds(10);
   if (UsingTestbench()) {
-    initial_delay = std::chrono::milliseconds(10);
+    initial_delay = kShortDelayForTestbench;
   }
 
+  auto constexpr kMaximumBackoffDelay = std::chrono::minutes(5);
+  auto constexpr kBackoffScalingFactor = 2.0;
   return ExponentialBackoffPolicy(initial_delay,
-                                  /*maximum_delay=*/std::chrono::minutes(5),
-                                  /*scaling=*/2.0)
+                                  /*maximum_delay=*/kMaximumBackoffDelay,
+                                  /*scaling=*/kBackoffScalingFactor)
```

Comment:
The `/*argname=*/` comments now seem less useful.

Reply:
Fixed.